### PR TITLE
Re-adding discussion of version control vs. online doc editing w/ intro para by @tracykteal.

### LINKED ISSUE #180: Keep detailed analysis of double recommendation for manuscripts?
See line 1205++ in manuscript - please add a comment to this issue saying whether you want to keep the detailed description or drop it.

### PR DESCRIPTION
**Pull request by gvwilson**
This PR also turns a couple of larger text samples into figures and inserts page breaks to
prevent widowed headers.

Closes #180.

### CODE REVIEW

**Comment by jennybc** [comment]
LGTM

**Comment by jkitzes** [comment]
Thumbs up. Just two very small things -

1. It's now been 3 years since publication of the best practices paper, not 2 (as in the first word of the paper). 
2. In the reference to my website, 6 in the ref list, there's an extra `\discretionary{-}{}{}` rendering in the URL.

Otherwise looks good to me!

**Comment by gvwilson** [comment]
Thanks @jkitzes - both fixed.

**Comment by lexnederbragt** [comment]
LGTM too

File: good-enough-practices-for-scientific-computing.tex [hunk 1]
```diff
@@ -1237,68 +1251,59 @@ \subsection*{Text-based Documents Under Version Control}
...
-
-Collaborative editing in something like Google Docs is often therefore
-an effective approach. This solution meets the requirements that we've
-outlined above, although you do lose some benefits of text-based
-formats (notably, being able to use non-proprietary tools for editing,
-sharing, and viewing and being amenable to version control). We still
-recommend \emph{against} traditional desktop tools like LibreOffice
-and Microsoft\textregistered Word because they make collaboration more
-difficult than necessary.
+\subsection*{Discussion}
```
Comment (lexnederbragt) [inline good-enough-practices-for-scientific-computing.tex:1254]: Calling this subsection 'Discussion' may throw of readers in thinking this is the discussion section of the entire paper. In the first submission, this section was called "Why Two Recommendations for Manuscripts?”. Can we keep that?

Comment (lexnederbragt) [reply good-enough-practices-for-scientific-computing.tex:1254]: see #187

Comment (tracykteal) [reply good-enough-practices-for-scientific-computing.tex:1254]: +1 to this point

**Comment by tracykteal** [comment]
LGTM

**Pull request merged**